frontend/src/components/viewers/MetricsVisualizations.tsx

```diff
@@ -36,44 +37,60 @@ interface MetricsVisualizationsProps {
  * and multiple visualizations associated with one artifact.
  */
 export function MetricsVisualizations({ artifacts, artifactTypes }: MetricsVisualizationsProps) {
-  // system.ClassificationMetrics contains confusionMatrix or confidenceMetrics.
-  // TODO: Visualize confusionMatrix using system.ClassificationMetrics artifacts.
-  // https://github.com/kubeflow/pipelines/issues/5668
-  let classificationMetricsArtifacts = filterArtifactsByType(
-    'system.ClassificationMetrics',
-    artifactTypes,
-    artifacts,
-  );
-
-  // There can be multiple system.ClassificationMetrics artifacts per execution.
-  // Get confidenceMetrics and confusionMatrix from artifact.
+  // There can be multiple system.ClassificationMetrics or system.Metrics artifacts per execution.
+  // Get scalar metrics, confidenceMetrics and confusionMatrix from artifact.
   // If there is no available metrics, show banner to notify users.
   // Otherwise, Visualize all available metrics per artifact.
-  const metricsAvailableArtifacts = getMetricsAvailableArtifacts(classificationMetricsArtifacts);
+  const verifiedClassificationMetricsArtifacts = getVerifiedClassificationMetricsArtifacts(
+    artifacts,
+    artifactTypes,
+  );
+  const verifiedMetricsArtifacts = getVerifiedMetricsArtifacts(artifacts, artifactTypes);
 
-  if (metricsAvailableArtifacts.length === 0) {
+  if (
+    verifiedClassificationMetricsArtifacts.length === 0 &&
+    verifiedMetricsArtifacts.length === 0
+  ) {
     return <Banner message='There is no metrics artifact available in this step.' mode='info' />;
   }
 
   return (
     <>
-      {metricsAvailableArtifacts.map(artifact => {
+      {verifiedClassificationMetricsArtifacts.map(artifact => {
         return (
           <React.Fragment key={artifact.getId()}>
             <ConfidenceMetricsSection artifact={artifact} />
             <ConfusionMatrixSection artifact={artifact} />
           </React.Fragment>
         );
       })}
+      {verifiedMetricsArtifacts.map(artifact => {
+        return (
+          <React.Fragment key={artifact.getId()}>
+            <ScalarMetricsSection artifact={artifact} />
+          </React.Fragment>
+        );
+      })}
```

Comment:
Usually, there are fewer visual space needed for scalar metrics. Shall we show them on the top?

By the way, this piece of code can be simplified (key can be added to any element):
```suggestion
      {verifiedMetricsArtifacts.map(artifact => 
        <ScalarMetricsSection artifact={artifact} key={artifact.getId()} />    
      })}
```

Reply:
Scalar metrics might need more spaces which push the remaining metrics to lower (because it is paged table, it will request an area of full page on Visualization Tab.)

Sounds good on simplifying key location!

Reply:
I see, you can decide!